src/components/shared/ItemDescription.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+class HandleDescLength extends React.Component {
```

Comment:
Rename component to 
`ExpandableText` or something better

Reply:
I've removed this one